extra/alacritty.man

```diff
@@ -32,10 +32,12 @@ Increases the level of verbosity (the max level is \fB\-vvv\fR)
 Prints version information
 .SH "OPTIONS"
 .TP
-\fB\-\-class\fR <instance> | <instance>,<general>
+\fB\-\-class\fR <general> | <general>,<instance>
 Defines the window class hint on Linux [default: Alacritty,Alacritty]
 
-On Wayland the instance class sets the `app_id`, while the general class is ignored.
+When only general class was passed it'll set instance to the same value.
+
+On Wayland the instance class sets the `app_id`, while the instance class is ignored.
```

Comment:
On Wayland the general class sets the `app_id`, right? That's why `--class whatever` wasn't working?

Reply:
Right.

Reply:
Seems like you didn't fix this?

Reply:
I did, at least I've applied you suggestion. Maybe github bugged for you?

Reply:
> On Wayland the instance class sets the `app_id`

It's the general class, not the instance class.

Also I didn't make a suggestion?